packages/react/src/hocs/CorbadoProvider.tsx

```diff
@@ -22,6 +22,16 @@ const CorbadoProvider: FC<CorbadoProviderProps> = ({ children, ...config }) => {
     return themesCleanup;
   }, []);
 
+  useEffect(() => {
```

Comment:
Do we still need the useEffect above if these two new useEffects react to changes already?

Reply:
nice point.. removing this

Reply:
removed and tested